### DISCUSSION TITLE
SBFspot send email

### DISCUSSION THREAD

**Discussion by shadowbyte68** [SBFspot/SBFspot#621] [Q&A]
SBFspot sends an email every day at 5:55 AM. Where can this be deactivated?

**Comment by sbf-** [comment]
SBFspot doesn´t send e-mail

**Comment by shadowbyte68** [comment]
I found an entry in crontab that was entered automatically, I have now deactivated it

` #55 05 * * * /usr/local/bin/sbfspot.3/monthdata
`
